dandi/metadata.py

```diff
@@ -105,7 +108,7 @@ def get_metadata(
                 pass
             else:
                 meta[key] = value
-    elif path.endswith((".NWB", ".nwb")):
+    if path.endswith((".NWB", ".nwb")):
```

Comment:
note: I thought to suggest to move this above so we allowed BIDS metadata to overload what is in .nwb but I think it is not yet decided matter: see discussion https://github.com/bids-standard/bids-specification/pull/761/files

Reply:
both get triggered since it's not `elif`, so we overload the metadata in any case. Are you saying you'd rather overload NWB with BIDS than BIDS with NWB? I actually think the latter (i.e. what we do now) is better since the scope of BIDS would be more general and that of NWB more specific.

Reply:
yeah - I meant the overloading in the other order, but as I have mentioned, I am ok with as is for now. Let's mark this one resolved